react/SelectionBar/index.jsx

```diff
@@ -23,13 +23,9 @@ actions = {
 
 */
 
-const SelectionBar = ({
-  t,
-  actions,
-  selected,
-  hideSelectionBar,
-  breakpoints: { isMobile, isTablet }
-}) => {
+const SelectionBar = ({ actions, selected, hideSelectionBar }) => {
+  const { t } = useI18n()
+  const { isMobile, isTablet } = useBreakpoints()
```

Comment:
Using useBreakpoins() is a BC if the APP doesn't use a `BreakpointsProvider`

Reply:
Thanks, I've edited https://github.com/cozy/cozy-ui/pull/1607/commits/b97fd3cefabd616ed289eb25f3346fe09957b1dd to add a BREAKING CHANGE mention.